Add a quick guide (#122)
owncloud/docs-ocis: modules/ROOT/pages/deployment/index.adoc

= Deployment
:toc: right

:description: There are several deployment options for ownCloud Infinite Scale (oCIS) available. Consider which one serves your purposes best. Generally speaking, you can decide between using Docker containers or manually set up a regular server.

// https://owncloud.dev/ocis/deployment/

{description}

> * The section xref:general/general-info.adoc[General Info] summarizes general and important information which is not dependent on the type how you install oCIS. Any setup refers to this general information as prerequisit knowledge without explicit reference.

Reviewer: ```suggestion
* The section xref:general/general-info.adoc[General Info] summarizes general and important information which is not dependent on the way that you install oCIS. Any setup refers to this general information as prerequisite knowledge.
```

* If you only want to play around and see what oCIS has to offer, we suggest the quick and easy xref:deployment/binary/binary-setup.adoc[Binary Setup] section.

> * In case you have already decided on a container instance with one or more containers, checkout the xref:deployment/container/container-setup.adoc[Container Setup] section.

Reviewer: ```suggestion
* If you have already decided on a container instance with one or more containers, checkout the xref:deployment/container/container-setup.adoc[Container Setup] section.
```
